client/pages/StatUnits/View/tabs/ContactInfo.jsx

```diff
@@ -406,7 +406,8 @@ class ContactInfo extends React.Component {
             <Grid.Row>
               <Grid.Column width={16}>
                 <label className={styles.boldText}>{localize('PersonsRelatedToTheUnit')}</label>
-                <PersonsList name="persons" value={data.persons} localize={localize} readOnly />
+                {/* The rendering of this component does not work currently
```

Comment:
Do you have an idea on why it fails?

Reply:
@jhf Not yet, React is whining about the export of PersonsList. Will look into it further tomorrow

Reply:
Maybe you can use ChatGpt to review the related code and look for import/export mismatches?

Reply:
It's related to the export of the PersonsList in React, we need to figure out why React is whining about it.

Reply:
The problem was missing "export" in the PersonsList component and improper import of the component.